virttest/virt_vm.py

```diff
@@ -638,6 +638,8 @@ def get_address(self, index=0):
         :raise VMAddressVerificationError: If the MAC-IP address mapping cannot
                 be verified (using arping)
         """
+
+        flexible_index = self.params.get("flexible_nic_index", 'no')
```

Comment:
@sathnaga 
According to this solution, my suggestion is make ```flexible_index``` as the 3rd param of this function:
```
     def get_address(self, index=0, flexible_index=False):
```
And use ```index=0``` as high priority